Resources/Prototypes/Entities/Mobs/Cyborgs/base_borg_chassis.yml

```
@@ -32,7 +32,7 @@
         - MobLayer
   - type: MovementSpeedModifier
     baseWalkSpeed : 2.5
-    baseSprintSpeed : 4.5
+    baseSprintSpeed : 4.8
```

Revert to 4.5 for direction. Also add in friction here from this [comment](https://github.com/DeltaV-Station/Delta-v/pull/4819/files#r2579077343). 

```suggestion
    baseSprintSpeed : 4.5
    baseWeightlessModifier: 0.75 # DeltaV
    baseWeightlessAcceleration: 1.25 # DeltaV
```